packages/babel-parser/test/helpers/to-fuzzed-options.js

```diff
@@ -0,0 +1,111 @@
+/* eslint-disable no-confusing-arrow */
+
+const random = (min, max) => Math.floor(Math.random() * (max - min + 1) + min);
+const clone = value => JSON.parse(JSON.stringify(value));
+const { FUZZ } = process.env;
+
+const toDescriptorAssignedObject = (delta, object) =>
+  delta.reduce(
+    (object, [key, descriptor]) => (
+      !descriptor || descriptor.delete
+        ? delete object[key]
+        : Object.defineProperty(object, key, descriptor),
+      object
+    ),
+    clone(object),
+  );
+
+const toAdjust = adjustments =>
```

Comment:
Nit: `toAdjustFunction`, it makes it a bit clear to understand what this should do.

Reply:
Done: https://github.com/babel/babel/pull/14201/commits/5e760114b5f1650507e12ecec7c605ec74abf80b